docs/development/manufacturer/manufacturer-design-guidelines.md

```diff
@@ -166,14 +166,41 @@ For connector pinout please refer to the [Betaflight Connector Standard](connect
 
 :::note
 
 We do not recommend using the Bosch BMI-270 IMU, because its gyroscope is uncalibrated. As a result, when gyro is integrated to return a change in attitude, the new attitude estimate can be in error, sometimes as much as 5% or 10%. This causes an angle offset until the accelerometer data can be used.
 
 :::
 
+
+
+### 3.1.2.1 Future IMU Options and How to Select Preferred Options
+
+As the MPU-6000 is EOL, the currently recommended IMU for most applications is the TDK InvenSense ICM-42688-P. This gyro has been proven in many designs to provide excellent performance with low noise and good durability. In all cases, it is strongly recommended that the gyro be powered from its own LDO. For the ICM-42688-P, this is *required*. See below for more on electrical noise considerations. 
+
+The ICM-42688-P also supports an external clock input, which can yield increased stability and performance. If your design allows, consider adding external clock support. See the following PR for more info: https://github.com/betaflight/betaflight/pull/13912
+
+Future IMU selection should be carried out with close involvement of the Betaflight development group. Early hardware validation samples should be explored in collaboration with Betaflight developers to determine the suitability of these IMU units in relevant environments.
+The ability to customize IMU lowpass filtering and operate within the same GRMS/Shock environment allows for maximum portability of existing filtering and tune schemes, but this development must occur with complete hardware samples and flown in representative flight regimes in order to replicate the EMI environment end users will experience.
+
+The IMU sensors, designed for applications outside of sUAS, are typically subjected to very harsh electromagnetic environments. Ensuring electromagnetic compatibility when using these immediately adjacent to ultrasonically switched power MOSFET devices, constantly operational radio frequency devices (such as remote control and FPV video systems), under thermal stresses of moving over 1kW through the complete flight stack, are a nontrivial operation. In order to minimize risks of flyaway and brownout behaviors which can be observed if IMU data filtering and power delivery are inadequate, proper circuit design and validation testing must be performed.
+
+:::note
+
+#### Advised power supply design for ICM-42xxx IMU. (also applies to others)
+
+These IMU require a stable and clean power supply to function correctly, provided with this they are very capable IMU for flight controller use.  
+Peak to peak noise of under 50uV on the supply should be the ideal goal.  
+Considerations for a suitable LDO are at least 500mA rated output allowing for extra capacitors on the output, two 20nF.  
+A PSRR >70dB down as low as 1Hz is preferred.  
+Where space allows, a dedicated LDO and circuit is advised for the IMU.
```

Comment:
```suggestion
Where space allows, a dedicated LDO and circuit is advised for the IMU. For ICM-42688-P this is *required*
```

Reply:
@haslinghuis , add a period, and i will click commit.